proto/metadata.proto

```diff
@@ -816,6 +816,9 @@ message Metadata {
     // Juniper b/419536104
     bool config_leaf_create_required = 291;
 
+    // Devices does not support static mpls lsp
+    bool static_mpls_lsp_oc_unsupported = 292;
```

Comment:
looks like a duplicate of existing deviation `bool static_mpls_unsupported = 283;` at 
https://github.com/openconfig/featureprofiles/pull/4131/files#diff-4891c0dff9aa45c82082583dfb4f85fa166c328f493edd8adff75f6a84e576aaR791

Reply:
Ah, thank you for pointing that out, you're correct.  But thanks for changing it here.